llama.cpp

```diff
@@ -4965,9 +4970,7 @@ static bool llm_load_tensors(
                             model.output = ml.create_tensor(ctx_output_split, tn(LLM_TENSOR_OUTPUT, "weight"), {n_embd, n_vocab}, false);
                             // if output is NULL, init from the input tok embed
                             if (model.output == NULL) {
-                                model.output = ml.create_tensor(ctx_output, tn(LLM_TENSOR_TOKEN_EMBD, "weight"), {n_embd, n_vocab});
-                                ml.n_created--; // artificial tensor
-                                ml.size_data += ggml_nbytes(model.output);
+                                model.output = ml.create_tensor(ctx_output, tn(LLM_TENSOR_TOKEN_EMBD, "weight"), {n_embd, n_vocab}, true);
```

Comment:
Was the intention to make the `duplicated` argument `true`? I assume that yes, because then the old behavior would be kept. 

This seems like `required` is set to `true` and leaves `duplicated` to `false`.

(this also applies to the other places where `model.output` is initialized from the token_embd tensor)

```suggestion
                                model.output = ml.create_tensor(ctx_output, tn(LLM_TENSOR_TOKEN_EMBD, "weight"), {n_embd, n_vocab}, true, true);
```